test/README.md

```diff
@@ -6,29 +6,43 @@ Thanks for writing tests! Here's a quick run-down on our current setup.
 
 Please familiarise yourself with these if you plan on contributing! :+1:
 
- - [enzyme](https://github.com/airbnb/enzyme)
- - [mocha](https://mochajs.org)
- - [chai](http://chaijs.com)
- - [jsdom](https://github.com/tmpvar/jsdom)
-
+- [enzyme](https://github.com/airbnb/enzyme)
+- [mocha](https://mochajs.org)
+- [chai](http://chaijs.com)
+- [jsdom](https://github.com/tmpvar/jsdom)
+- [docker](https://github.com/docker/docker)
 
 ## Commands
 
-##### Run the core mocha unit/integration test suite.
+### Run the core mocha unit/integration test suite.
 `npm run test`
 
-#### Watch the core mocha unit/integration test suite.
+### Watch the core mocha unit/integration test suite.
 `npm run test:watch`
 
-#### Run the mocha test suite using the karmer runner.
+### Run the mocha test suite using the karmer runner.
 `npm run test:karma`
 
-#### Run the e2e selenium tests.
+### Run the e2e selenium tests.
 `npm run test:e2e`
 
-#### Run the visual regression tests
+### Run the visual regression tests
 `npm run test:regressions`
 
```

Comment:
Might be good to have a brief description of what each of these does. "Run the mocha test suite using the karmer runner." What does that even mean? :smile: